cczoo/cross_lang_framework/install cr_lan framework .bash

```diff
@@ -0,0 +1,164 @@
+function version_lt() { test "$(echo "$@" | tr " " "\n" | sort -rV | head -n 1)" != "$1"; }
+function month()
+{
+local j=0
+for i in Jan Feb Mar Apr May Jun Jul Aug Sep Oct Nov Dec
+    do
+        let j+=1;
+        if [[ $1 = $i ]];then
+
+            return $j
+            break
+        else
+            continue
+        fi
+    done
+}
+echo "begin to judge whether the GCC,kernel,OS satisfy requirements"
+kernel_version_base="5.11.0"
+gcc_version_base="7.5.0"
+os_version_base="18.04"
+kernel_v=$(uname -r)
+kernel_version=${kernel_v%%-*}
+echo $kernel_version
+system_OS=$(awk -F= '/^NAME/{print $2}' /etc/os-release | sed 's/\"//g')
+echo $system_OS
+OS="Ubuntu"
+if [ "$system_OS" != $OS ]; then
+        echo "Cross_language_framework can only be used in Ubuntu"
+        exit 1
+fi
+os_version=$(cat /etc/issue| sed '1!d' | sed -e 's/"//g' | awk '{print $2}')
+if version_lt $os_version $os_version_base; then
+        echo "Cross_language_framework can only be used in Ubuntu with version>=18.04"
+        exit 1
+fi
+gcc_version=$(gcc --version |sed '1!d' | sed -e 's/"//g' | awk '{print $4}')
+echo $gcc_version
+if version_lt $gcc_version $gcc_version_base; then
+        echo "gcc version dose not satisfy requirement, please use gcc version>=7.5.0"
+        exit 1
+fi
+if version_lt $kernel_version $kernel_version_base; then
+        echo "kernel version dose not satisfy requirement, please use kernel version>=5.11.0"
+        exit 1
+fi
+#install SGX SDK
+echo " begin to install SGX SDK Prerequisites"
+apt-get update
+apt-get install -y libssl-dev libcurl4-openssl-dev libprotobuf-dev
+apt-get install -y build-essential python
+apt-get install -y autoconf gawk bison
+apt-get install -y libprotobuf-c-dev protobuf-c-compiler
+apt-get install -y python3-protobuf
+apt-get install -y unzip
+if [[ -d "/opt/intesl/sgxsdk/" ]]; then
+  echo "SGX SDK has been installed"
+elif [[   $os_version == *"20.04"* ]]; then
+  echo "begin to download SGX SDK"
+  wget https://download.01.org/intel-sgx/latest/dcap-latest/linux/distro/ubuntu20.04-server/sgx_linux_x64_sdk_2.17.100.3.bin
```

Comment:
The potential issue will be that if there is one new version SDK, the SDK version will not match so as to download failed here.